src/pyhf_benchmark/manager.py

```diff
@@ -11,7 +12,7 @@ def __init__(self, meta=None):
         self._start_time = time.time()
         self.times = 0
         self.directory = Path(
-            f"output/run_{time.strftime('%Y%m%d', time.localtime())}_{int(self._start_time)}"
+            f"{pyhf_benchmark.__path__[0]}/../../output/run_{time.strftime('%Y%m%d', time.localtime())}_{int(self._start_time)}"
```

Comment:
You don't want to do this as this will give the wrong behavior. You probably want to define the output directory relative to the directory where `pyhf-benchmark` is _run from_ not from _where it installed_.